### PR TITLE
Simplier instructions and Language adjustments in /developer-guides/quick-start

### PR DESCRIPTION
**Pull request by Rodriq**

### CODE REVIEW

**Comment by Sing-Li** [comment]
This entire page is being revised  - to ensure the procedure is working for 1.0 builds.

The existing instructions are totally broken, as verified by 10s of GSoC 2019 students trying to build.

Please rebase your modifications AFTER our revisions.

**Comment by Rodriq** [comment]
@Sing-Li updated

File: developer-guides/quick-start/README.md [hunk 1]
```diff
@@ -19,27 +20,19 @@ You may notice build WARNINGs related to _peer dependencies_  or other transitiv
...
     There is no need to install `node` or `npm`, as meteor already includes them.  Verify by:
-
     `meteor node -v`
-
     `meteor npm -v`
 
 3. Get rocket.chat code
-
-    `git clone https://github.com/RocketChat/Rocket.Chat.git`
+- `git clone https://github.com/RocketChat/Rocket.Chat.git` (if you wish to clone your own fork, replace with your repo link which may look like __*https://github.com/your-user-name/Rocket.Chat.git*__).
```
Comment (Sing-Li) [inline developer-guides/quick-start/README.md]: please revert this.   it adds complexity.

Suggest to add here ->
    Get rocket.chat code (or clone from your own fork)

File: developer-guides/quick-start/README.md [hunk 2]
```diff
@@ -19,27 +20,19 @@ You may notice build WARNINGs related to _peer dependencies_  or other transitiv
...
-
-    `cd Rocket.Chat`
-
     `meteor npm install`
 
 5. Start building (can take 5 minutes, please be patient)
-
-    `meteor npm start`
-
+- `meteor npm start` (Warning: It may take a while to build so please be patient).
```
Comment (Sing-Li) [inline developer-guides/quick-start/README.md:37]: not necessary at all, please revert

already mentioned in heading 5

**Review by Sing-Li — CHANGES_REQUESTED** [review]

File: developer-guides/quick-start/README.md (hunk 1 above)
Comment (Rodriq) [reply developer-guides/quick-start/README.md]: ok

File: developer-guides/quick-start/README.md (hunk 2 above)
Comment (Rodriq) [reply developer-guides/quick-start/README.md:37]: true

**Review by Rodriq — COMMENTED** [review]
Hope it's better now.

**Review by Sing-Li — APPROVED** [review]

**Review by Faria1212 — APPROVED** [review]

**Comment by MartinSchoeler** [comment]
Hey @Rodriq I will be closing this PR, most of the changes done in this pr has been already done in the master branch. Thanks! I

**Pull request closed without merging**